File: JavaScript/Array 2D y 3D/Recorrido de matriz 3D.js
const matriz3d = [
    [[1, 2], [3, 4]],
    [[5, 6], [7, 8]]
];

console.log("---Recorrido de matriz 3D---");
for (let i = 0; i < matriz3d.length; i++) {
    console.log("Capa " + i + ":");
    for (let j = 0; j < matriz3d[i].length; j++) {
        let fila = "";
        for (let k = 0; k < matriz3d[i][j].length; k++) {
            fila += matriz3d[i][j][k] + " ";
        }
        console.log(fila);
    }
}

File: Python/Array 2D y 3D/Recorrido de matriz 3D.py
matriz3d = [
    [[1, 2], [3, 4]],
    [[5, 6], [7, 8]]
]

print("---Recorrido de matriz 3D---")
for i in range(len(matriz3d)):
    print("Capa", i, ":")
    for j in range(len(matriz3d[i])):
        for k in range(len(matriz3d[i][j])):
            print(matriz3d[i][j][k], end=" ")
        print()